cmd/lncli/commands.go

```
@@ -373,6 +373,22 @@ func sendCoins(ctx *cli.Context) error {
 			"sweep all coins out of the wallet")
 	}
 
+	client, cleanUp := getClient(ctx)
+	defer cleanUp()
+	minConfs := int32(ctx.Uint64("min_confs"))
+
+	// In case that the user has specified the sweepall flag, we'll
+	// calculate the amount to send based on the current wallet balance.
+	if ctx.Bool("sweepall") {
+		balanceResponse, _ := client.WalletBalance(
```

We shouldn't swallow the error here but instead return it.